src/30_Advanced/SeatMap.html

```diff
@@ -0,0 +1,224 @@
+        background-color: white;
+        border-radius: 3px;
+      }
+      .opaque {
+        opacity: 0.5;
+        filter: alpha(opacity=50);
+      }
+      #choose-seats-button {
+        padding-bottom: 15px;
+        padding-right: 10px;
+      }
+      .seat-map-svg {
+        box-sizing: border-box;
+        padding-top: 30px;
+      }
+      #pan-zoom {
+        display: initial;
+      }
+      #pan-zoom > svg {
+        left: 0;
+        right: 0;
+        margin: auto;
+      }
+      .hide {
+        display: none;
+      }
+      .list-container {
+        position: relative;
+        height: 70vh;
+      }
+      .ampstart-card {
+        min-width: 100%;
+        padding: 1rem;
+      }
+    </style>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style>
+    <noscript>
+      <style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style>
+    </noscript>
+
+  </head>
+  <body>
+
+    <!-- Here we are using an `amp-state` called `seatmap`.
+      `scale` variable is used to keep track of the scale of the content being zoomed using by `amp-pan-zoom`.
+      `zoom` variable is used to keep track of the seat map is in a zoomed state or not. This variable is used to
+    decide whether to show a + or - button.
+    `selectedSeats` variables keeps a  list of selected seat ids.  -->
+    <amp-state id="seatmap">
+      <script type="application/json">
+        {
+          "scale": 1,
+          "selectedSeats": [],
+          "zoom": 0
+        }
+      </script>
+    </amp-state>
+
+    <h1>Seat map</h1>
+    <!-- You can implement a seat map by using `amp-list`, `amp-bind` and `amp-pan-zoom` where
+    `amp-list` loads the list of seats. 
+    We are using the `transformEnd` action from [amp-pan-zoom](https://www.ampproject.org/docs/reference/components/amp-pan-zoom)
+    to update the value of transform.scale variable.
+    We display 2 types of seats: wheelchair and standard. Each seat can be available, unavailable and selected. Everytime a seat is 
+    selected we add the seat id to the `seatmap.selectedSeats` variable.
+    -->
+     <div class="list-container">
+     <amp-list layout="fill" src="/json/seats.json" items="." single-item noloading>
+      <div placeholder>
+        <svg class="opaque disabled-seat seat-map-svg" viewBox="0 0 [[.Width]] [[.Height]]" preserveAspectRatio="xMidYMin slice">
```

Comment:
The source code view does not render the template - is this WAI?

Reply:
It's rendering it for me